Services/DbService.js

```diff
@@ -0,0 +1,15 @@
+const MongoClient = require('mongodb').MongoClient
+
+const url = 'mongodb://root:password@localhost:27017'
+const dbName = 'restaurant'
+
+const Client = new MongoClient(url, {useNewUrlParser: true, useUnifiedTopology: true})
+
+let connectToDb = (cb) => {
+    Client.connect((err) => {
+        let db = Client.db(dbName)
+        cb(db)
+    })
+}
+
+module.exports = connectToDb
```

Comment:
Not sure how this module is on your branch. Please delete to avoid merge conflicts.